app/schedule/serializer.js

```diff
@@ -0,0 +1,5 @@
+import JSONAPISerializer from 'ember-data/serializers/json-api';
```

Comment:
the convention for ember data is to import DS and read the classes off of that. so do
```javascript
import DS from 'ember-data';

export default DS.JSONAPISerialzer({
  ...
});
```